src/Workspaces/Core/Portable/Workspace/Workspace_Editor.cs

```diff
@@ -435,7 +435,7 @@ private ISet<DocumentId> GetProjectOpenDocuments_NoLock(ProjectId project)
                     // Note: we pass along the newText here so that clients can easily get the text
                     // of an opened document just by calling TryGetText without any blocking.
                     currentSolution = oldSolution.WithDocumentTextLoader(documentId,
-                        new ReuseVersionLoader(oldDocument.State, newText), newText, PreservationMode.PreserveIdentity);
+                        new ReuseVersionLoader((DocumentState)oldDocument.State, newText), newText, PreservationMode.PreserveIdentity);
```

Comment:
are you asking about casting? ReuseVersionLoader expects DocumentState and oldDocumet.State used to be return DocumentState, so I didn't change any semantic.